### PR TITLE
ISPN-14492 Java 17 instead of 11

### PR DESCRIPTION
**Pull request by karesti**
https://issues.redhat.com/browse/ISPN-14492

Backport #10613

### CODE REVIEW

**Comment by tristantarrant** [comment]
Please update the contribution guide too.

**Comment by karesti** [comment]
@tristantarrant guide updated

File: documentation/src/main/asciidoc/topics/ref_quick_start.adoc [hunk 1]
```diff
@@ -8,7 +8,7 @@ management tooling, and either the {brandname} server or core libraries.
 
 To run {brandname}, you'll need:
 
-* Java 11 JDK at a minimum. {brandname} but also works with JDK 17, 18 and 19.
+* Java 17 JDK at a minimum. {brandname} also works with 18, 19.
```
Comment (tristantarrant) [inline documentation/src/main/asciidoc/topics/ref_quick_start.adoc:11]: We will still build for JDK 11

Comment (karesti) [reply documentation/src/main/asciidoc/topics/ref_quick_start.adoc]: fixed again

File: documentation/src/main/asciidoc/topics/proc_adding_hotrod_client_dependencies.adoc [hunk 2]
```diff
@@ -4,7 +4,7 @@ Add Hot Rod Java client dependencies to include it in your project.
 
 .Prerequisites
 
-* Java 8 or Java 11
+* Java 8, Java 11 or Java 17
```
Comment (pruivo) [inline documentation/src/main/asciidoc/topics/proc_adding_hotrod_client_dependencies.adoc:7]: I believe we dropped Java 8 in ISPN 14+
```suggestion
* Java 11 or greater.
```

File: documentation/src/main/asciidoc/topics/ref_quick_start.adoc [hunk 3]
```diff
@@ -8,7 +8,7 @@ management tooling, and either the {brandname} server or core libraries.
 
 To run {brandname}, you'll need:
 
-* Java 11 JDK at a minimum. {brandname} but also works with JDK 17, 18 and 19.
+* Java 11 JDK at a minimum. {brandname} also works with 17, 18, 19.
```
Comment (pruivo) [inline documentation/src/main/asciidoc/topics/ref_quick_start.adoc:11]:
```suggestion
* Java 11 JDK at a minimum.
```
AFAIK, we don't test with JDK 18/19. Speaking of testing, we may need a profile to test with JDK 11, right? 
cc @tristantarrant wdyt? is it worth adding such complexity?

**Review by pruivo — CHANGES_REQUESTED** [review]

**Comment by karesti** [comment]
@pruivo I added your changes

**Review by pruivo — APPROVED** [review]

**Comment by tristantarrant** [comment]
Merged, thanks

**Pull request merged**